### DISCUSSION TITLE
Support for text size from iOS

### DISCUSSION THREAD

**Discussion by mattstratton** [astashov/liftosaur#171] [Ideas]
In iOS (I assume in android too?) you can set the text size modifier both globally and per-app. Would be great to be able to have Liftosaur use that for those of us with less than amazing eyesight (the text size is kind of small for me without my reading glasses)

**Comment by astashov** [comment]
I think on Android it actually respects system font size...

**Comment by astashov** [comment]
There's a setting for that now
